### DISCUSSION TITLE
Workspaces won't stay on after restaring, and sometimes come back after turning the setting on and restarting. It's very unstable for some reason.

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#437] [Q&A]
I'm not sure how to write this as a bug report so I'm writing it here. There's no one definite way to reproduce it, the behavior with workspaces and Floorp is really unstable. I described what happens in the title, and it still happens even if I turn all my addons off. Even if it I do have it on and working, if I restart Floorp then its gone. I don't know what to do.

**Comment by menndouyukkuri** [comment]
If there is only one "default" workspace when updating, it may be disabled.
Is it different from that?

**Comment by Ryosuke-Asano** [comment]
Maybe fixed

**Reply by YousufSSyed** [reply]
I'm trying Floorp againk, will bookmark this, and see if the issue is still there.

**Reply by YousufSSyed** [reply]
@surapunoyousei Yeah this still has issues. When I try clicking the workspace button on the top left corner, it doesn't show the workspace manager. When I open it in a new window. It looks like I'm going to have to wait a while for Floorp (non-beta version) to become more stable, it's still pretty buggy.

**Reply by Ryosuke-Asano** [reply]
Please paste video

**Reply by YousufSSyed** [reply]
I just tried the workspaces button again and its working now. I'll see if this still persists.
